dashboard/test/controllers/lti/v1/account_linking_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class Lti::V1::AccountLinkingControllerTest < ActionController::TestCase
+  setup do
+    @user = create(:teacher, email: 'test@lti.com')
+    @lti_integration = create :lti_integration
+  end
+
+  setup do
```

Comment:
Is the two `setup` blocks a Ruby best practices thing? One set's variables and one sets up stubs?

Reply:
Whoops, no I think I accidentally added a second block when I was debugging some DCDO-related errors. Will update this